cockroachcloud/use-managed-service-backups.md

```diff
@@ -101,6 +100,17 @@ For each table in the database, the following details display:
 
 - The number of **Rows** captured in the backup
 
+### Incomplete Backups
+
+To view any failed or pending backups, click the Incomplete Backups tab on your cluster's **Backups** page. 
+
+For each incomplete backup, the following details display:
+
+- **Started**: The date and time the backup began  
+- **Duration**: The amount of time the backup ran for 
+- **Status**: The error code and text for the failed backup. No error code is shown for pending backups  
```

Comment:
nit: "the failed backup" => "failed backups"? To match the form of "pending backups"?